[Doc Improvement][Share to Teams support in Gov clouds][3777626] (#10864)
MicrosoftDocs/msteams-docs: msteams-platform/concepts/build-and-test/share-to-teams-overview.md

Share to Teams is a button enabled on any web app, personal app, or tab that allows users to share content to other users in Teams.

> [!NOTE]

> > Share to Teams isn't available for [Government Community Cloud (GCC) High and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.

Author: Retaining

You can also decide the placement of the Share to Teams button on the app. Users can pin the crucial content and refer to it while collaborating with Teams.

:::image type="content" source="../../assets/images/share-to-teams/overview.PNG" alt-text="Screenshot showing the Share button on the app.":::

> You can enable this feature on the following scopes:

Reviewer: ```suggestion
You can enable Share to Teams button in the following:
```

Author: Updating as:

You can enable Share to Teams button in the following scopes:

* Web apps

> * Personal app or tab

Reviewer: Add the gov clouds note in Ln# 24, after the bullet points.
